test/JDBC/jdbc_schedule

```diff
@@ -71,6 +71,12 @@ ignore#!#sys_database_principals_dep_for_13_x-vu-verify
 ignore#!#sys_server_principals_dep_for_13_x-vu-cleanup
 ignore#!#sys_server_principals_dep_for_13_x-vu-prepare
 ignore#!#sys_server_principals_dep_for_13_x-vu-verify
+ignore#!#bbf_view_def-vu-prepare
+ignore#!#bbf_view_def-vu-verify
+ignore#!#bbf_view_def-before-14_5-vu-prepare
+ignore#!#bbf_view_def-before-14_5-vu-verify
+ignore#!#bbf_view_def-before-14_7-vu-prepare
```

Comment:
To minimize the difference between 2_X_DEV and 3_X_DEV, let's use the same name, i.e., bbf_view_def-before-14_7-or-15_2

Reply:
Done.